plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/DeployPreferencesModel.java

```diff
@@ -99,6 +102,14 @@ public void setStopPreviousVersion(boolean stopPreviousVersion) {
     this.stopPreviousVersion = stopPreviousVersion;
   }
 
+  public boolean isConfigDeploy() {
```

Comment:
Haven't tested, but for the data binding POJO thing to work, I think it's intentional to use this convention `isSomeValue()`.